source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1279,6 +1277,22 @@ def is_olympic_site(entry_point, instance):
     return False
 
 
+def is_olympic_site(entry_point, instance):
+    if not entry_point:
+        return False
+    if entry_point.embedded_type not in [type_pb2.POI, type_pb2.ADDRESS]:
+        return False
```

Comment:
I don't think you need to test here. The lines below already check for these conditions (`is_olympic_poi` check for `type_pb2.POI` and the rest of the function tests for `type_pb2.ADDRESS`).

Reply:
Done